### DISCUSSION TITLE
Problem displaying the default selected item when listing database records in Livewire

### DISCUSSION THREAD

**Discussion by mustafac1** [livewire/livewire#5591] [1. Help] [answered]
- laravel: 9.19
- livewire: 2.x

**Problem statement :**
- I have created a component that lists the data in the database with Livewire. In the component, I want the selected item by default to be marked as selected. However, I'm having an issue where the selected item is not displayed even though I'm sure the selected item is set correctly. I'm wondering what the problem is and how it can be fixed.

**Codes**
![image](https://user-images.githubusercontent.com/57503952/221959043-b056f3e4-49c4-427e-af58-7cd64b81166a.png)

**Web Browser**
![image](https://user-images.githubusercontent.com/57503952/221959359-96d5bdbb-04e5-4327-8e3f-abd4c9bfbaf1.png)

**MYSQL**
![image](https://user-images.githubusercontent.com/57503952/221960410-fdc01481-e968-4f81-aec1-ca43ffe717cc.png)

**Comment by joshhanley** [comment, marked as answer]
@mustafac1 as Livewire controls what values are selected based on the property it is bound to, adding `selected` to an option won't work as Livewire will overwrite it due to data binding. 

So you will need to set your property `$set_lang_id` with the initial value you want selected.

Hope this helps!
